Docs/developer_guide/build_instructions/macos.md

```diff
@@ -102,16 +140,43 @@ See list of issues common to all operating systems on [Common errors](common_err
 ### error while configuring PCRE: "cannot run C compiled program"
 
 If the XCode command line tools are not properly set up on macOS, PCRE could fail to build in the Superbuild process with the errors like below:
-```
+
+```bash
 configure: error: in `/Users/fedorov/local/Slicer4-Debug/PCRE-build':
 configure: error: cannot run C compiled programs.
 ```
 
 To install XCode command line tools, use the following command from the terminal:
-```
+
+```bash
 xcode-select --install
 ```
 
 ### dyld: malformed mach-o: load commands size (...) > 32768
 
-Path of source or build folder is too long. For example building Slicer in */User/somebody/projects/something/dev/slicer/slicer-qt5-rel* may fail with malformed mach-o error, while it succeeds in */sq5* folder. To resolve this error, move source and binary files into a folder with shorter full path and restart the build from scratch (the build tree is not relocatable).
+Path of source or build folder is too long. For example building Slicer in */User/somebody/projects/something/dev/slicer/slicer-qt5-rel* may fail with malformed mach-o error, while it succeeds in */opt/s* folder. To resolve this error, move source and binary files into a folder with shorter full path and restart the build from scratch (the build tree is not relocatable).
+
+
+### Packaging errors
+
+#### Fixing @rpath errors during packaging
+
+If an error like
+
+```
+warning: target '@rpath/QtGui.framework/Versions/5/QtGui' is not absolute...
+warning: target '@rpath/QtGui.framework/Versions/5/QtGui' does not exist...
+```
+
+is present during packaging - doublecheck that Slicer was built with Qt that was built from source and not Qt that was installed from a web-installer or homebrew.
+
+#### LibArchive pointing to a nonexistent path
+
+If a packaged Slicer is launched on another mac and it crashes with the error log saying that
+
+```
+Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib
+  Referenced from: Slicer.app/Contents/lib/Slicer-4.13/libarchive.17.dylib
+```
+
+It means that `libarchive` has has found homebrew versions of some of it's requirements, rather than local ones. For the packaged version of Slicer to run on other machines none of the prerequisites should be installed via homebrew. For example `lz4` and `zstd` are bundled with `subversion` and `rsync` so if you have these two application installed via homebrew, `libarchive` will grab them from `/usr/local/opt/` and the packaged Slicer will not run on other machines. The solution is either to remove them from from homebrew with `brew remove lz4` and `brew remove zsdt` or to change the `$PATH` so that the local build folder goes before `/usr/local/opt/`. After doing this Slicer should be rebuilt and repackaged.
```

Comment:
Is there a github issue filed for this?  would be good to file one is Slicer so we can look at how to force libarchive to use the right versions of these libraries.  If it means changing libarchive we can work to get it fixed upstream, like [this issue was](https://github.com/libarchive/libarchive/pull/1395).

Reply:
There wasn't an issue filed for this so I've [created one](https://github.com/Slicer/Slicer/issues/5415).

Reply:
Could you add a link to the issue to this paragraph?